tests/kernel_performance/install_qatestset.pm

```diff
@@ -152,6 +172,20 @@ sub run {
 
     select_console 'root-console' if (is_sle('16+') && get_var('HANA_PERF'));
 
+    if (has_selinux) {
+        my $selinux_mode = get_var('HANAPERF_SELINUX_SETENFORCE', 'Enforcing');
+        script_run('sestatus');
```

Comment:
Is it for debug purpose? If so, does it make sense to use script_output and publish it with record_info?